code/modules/admin/view_variables/debug_variables.dm

```diff
@@ -57,9 +93,12 @@
 		var/image/image = value
 		return "<a href='?_src_=vars;[HrefToken()];Vars=[REF(value)]'>[image.type] (<span class='value'>[get_appearance_vv_summary_name(image)]</span>) [REF(value)]</a>"
 
-	if(isfilter(value))
-		var/datum/filter_value = value
-		return "/filter (<span class='value'>[filter_value.type] [REF(filter_value)]</span>)"
+	// fun fact: there are two types of /filters. `/filters(/filters(), /filters(), ...)`
+	// isfilter() doesn't know if it's a parent filter(that has [/filters]s inside of itself), or a child filter
+	var/isfilter = isfilter(value)
+	var/is_child_filter = isfilter && !isdatum(owner) && !isappearance(owner) // 'child_filter' means each /filters in /atom.filters
+	if(is_child_filter)
+		return "/filters\[child\] (<span class='value'>[value:type]</span>)"
```

Comment:
why are we using `:` here?

Reply:
because `value` can be /datum and not-/datum(i.e. string, number), and then non-typed-variable makes sense, while to access a type from such non-typed-variable, `:` operator is a thing.
If I use `proc_param(datum/value)`, a value will be believed to have a datum.

Reply:
It's gone now